src/viser/_messages.py

```diff
@@ -65,6 +65,15 @@ class CameraFrustumMessage(Message):
     image_base64_data: Optional[str]
 
 
+@dataclasses.dataclass
+class GlTFMessage(Message):
```

Comment:
nit can we name this `GltfMessage`? I think this is slightly more consistent with other acronyms (like `GuiMessage`) in the repo

(also, should this be `GlbMessage`?)